packages/boot/test/bootstrapTests/ec-contract-upgrade.test.ts

```diff
@@ -0,0 +1,399 @@
+import { test as anyTest } from '@agoric/zoe/tools/prepare-test-env-ava.js';
```

Comment:
The name of this file is misleading. We're not upgrading any contracts here.

I suggest `ec-membership-update.test.ts` or the like.

Reply:
updated